components/motor/gpio/setup.go

```diff
@@ -30,15 +30,15 @@ type PinConfig struct {
 // Config describes the configuration of a motor.
 type Config struct {
 	Pins             PinConfig      `json:"pins"`
-	BoardName        string         `json:"board"`                   // used to get encoders
-	MinPowerPct      float64        `json:"min_power_pct,omitempty"` // min power percentage to allow for this motor default is 0.0
-	MaxPowerPct      float64        `json:"max_power_pct,omitempty"` // max power percentage to allow for this motor (0.06 - 1.0)
-	PWMFreq          uint           `json:"pwm_freq,omitempty"`
+	BoardName        string         `json:"board"`                    // used to get encoders
+	MinPowerPct      float64        `json:"min_power_pct,omitempty"`  // min power percentage to allow for this motor default is 0.0
+	MaxPowerPct      float64        `json:"max_power_pct,omitempty"`  // max power percentage to allow for this motor (0.06 - 1.0)
+	PWMFreq          uint           `json:"pwm_freq,omitempty"`       // TODO(RSDK-568): pwm_freq_hz?
```

Comment:
per @randhid, pwm frequency is generally understood to be in hertz

```suggestion
	PWMFreq          uint           `json:"pwm_freq,omitempty"`
```